### PR TITLE
Upgarde to quarkus v0.23.1

### LINKED ISSUE #209: Upgrade to Quarkus 0.23.0

### PR DESCRIPTION
**Pull request by lburgazzoli**
Fixes #209

### CODE REVIEW

**Comment by lburgazzoli** [comment]
Thx @oscerd for the initial work

File: integration-tests/mail/pom.xml [hunk 1]
```diff
@@ -26,7 +26,7 @@
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>camel-quarkus-integration-test-mail</artifactId>
-    <name>Camel Quarkus :: Integration Tests Mail :: Test</name>
+    <name>Camel Quarkus :: Integration Tests :: Mail :: Test</name>
```
Comment (ppalaga) [inline integration-tests/mail/pom.xml]: `<name>Camel Quarkus :: Integration Tests :: Mail</name>` maybe?

File: integration-tests/mail/pom.xml [hunk 2]
```diff
@@ -46,10 +46,6 @@
 <!--            <groupId>org.apache.camel.quarkus</groupId>-->
 <!--            <artifactId>camel-quarkus-integration-test-mail-mailbox</artifactId>-->
 <!--        </dependency>-->
-        <dependency>
-            <groupId>org.jvnet.mock-javamail</groupId>
-            <artifactId>mock-javamail</artifactId>
-        </dependency>
```
Comment (ppalaga) [inline integration-tests/mail/pom.xml]: This one is managed in integration-tests/pom.xml and should be removed from there too.

File: integration-tests/mail/pom.xml (hunk 1 above)
Comment (lburgazzoli) [reply integration-tests/mail/pom.xml]: done

File: integration-tests/mail/pom.xml (hunk 2 above)
Comment (lburgazzoli) [reply integration-tests/mail/pom.xml]: done

**Comment by lburgazzoli** [comment]
done

**Pull request merged**